atum/src/main/scala/za/co/absa/atum/utils/ExecutionPlanUtils.scala

```diff
@@ -97,11 +97,17 @@ object ExecutionPlanUtils {
     * @return The inferred output control measurements file path of the source dataset
     */
   def inferOutputInfoFileName(qe: QueryExecution, infoFileName: String = Constants.DefaultInfoFileName): Option[Path] = {
+    inferOutputInfoFileDir(qe).map { dir =>
+      new Path(dir, infoFileName)
+    }
+  }
+
+  private[atum] def inferOutputInfoFileDir(qe: QueryExecution): Option[String] = {
```

Comment:
Wouldn't it be better to make the function public, if used outside? It is useful in itself..

Reply:
Makes sense, done.